htdocs/web_portal/views/downtime/add_downtime.php

```diff
@@ -494,14 +505,41 @@ function validateUtcDates(){
     }
 
     function getSitesServices(){
-        var siteId=$('#Select_Sites').val();
-        if(siteId != null){ //If the user clicks on the box but not a specific row there will be no input, so catch that here
-            $('#chooseEndpoints').empty(); //Remove any previous content from the endpoints select list
-            $('#chooseServices').load('index.php?Page_Type=Downtime_view_endpoint_tree&site_id='+siteId,function( response, status, xhr ) {
-                if ( status == "success" ) {
-                    validate();
-                }
-            });
+        let siteID = $('#Select_Sites').val();
+
+        /**
+         * If the user clicks on the box but not a specific row
+         * there will be no input, so catch that here.
+         */
+        if (siteID != null) {
+            // Remove any previous content from the endpoints select list.
+            $('#chooseEndpoints').empty();
+
+            let urlPath;
+            const SE_FROM_QUERY_PARAMS = new URL(
+                this.location.href
+                ).searchParams.get('se');
+
+            if (SE_FROM_QUERY_PARAMS) {
+                /**
+                 * NOTE: If you use template literal for string interpolation,
+                 * i.e., `${}`. It has to be in one line.
+                 */
+                urlPath = 'index.php?Page_Type=Downtime_view_endpoint_tree' +
```

Comment:
why on this line are ' ' used when on the rest of the lines below ` ` is used?

Reply:
Hi @rowan04,
as it is in Draft. Can you please halt reviewing this PR, Please ? Apart from functionality review.

Reply:
ok, my bad :)

Reply:
* NOTE: If you use template literal for string interpolation,
 * i.e., `${}`. It has to be in one line.